lib/checks/tables/th-has-data-cells.js

```diff
@@ -41,19 +45,17 @@ return headers.reduce(function (res, header) {
 	// Look for any data cells or row headers that this might refer to
 	if (tableUtils.isColumnHeader(header)) {
 		hasCell = tableUtils.traverse('down', pos, tableGrid)
-		.reduce((out, cell) => {
-			return out || (cell.textContent.trim() !== '' &&
-				!tableUtils.isColumnHeader(cell));
-		}, false);
+		.reduce(dataCellIsNotEmpty, false) ||
+			tableUtils.traverse('up', pos, tableGrid)
+			.reduce(dataCellIsNotEmpty, false);
 	}
 
 	// Look for any data cells or column headers that this might refer to
 	if (!hasCell && tableUtils.isRowHeader(header)) {
 		hasCell = tableUtils.traverse('right', pos, tableGrid)
-		.reduce((out, cell) => {
-			return out || (cell.textContent.trim() !== '' &&
-				!tableUtils.isRowHeader(cell));
-		}, false);
+		.reduce(dataCellIsNotEmpty, false) ||
```

Comment:
The code called `tableUtils.isRowHeader` here before but the new function calls `isColumnHeader`. The tests still pass, but I wanted to make sure this wasn't an oversight